README.md

```diff
@@ -12,13 +12,13 @@ $ pip --version
 pip 19.2.3 from /usr/local/lib/python3.7/site-packages/pip (python 3.7)
 
 $ sam --version
-SAM CLI, version 0.22.0
+SAM CLI, version 0.19.0
```

Comment:
[Command 'sam local start-api' fails when using AWS::Include transform to add a Swagger file for 'DefinitionBody'](https://github.com/awslabs/aws-sam-cli/issues/1418)
> OS: MacOSX
sam --version: SAM CLI, version 0.21.0 (Fails)
sam --version: SAM CLI, version 0.19.0 (Successful)